CefSharp.Core/ManagedCefBrowserAdapter.cpp

```diff
@@ -78,7 +77,7 @@ void ManagedCefBrowserAdapter::CreateBrowser(BrowserSettings^ browserSettings, R
     CefString addressNative = StringUtils::ToNative(address);
 
     CefBrowserHost::CreateBrowser(window, _clientAdapter.get(), addressNative,
-        *browserSettings->_browserSettings, requestContext);
+        *browserSettings->_browserSettings, *requestContext);
```

Comment:
Was there an actual compiler error or were you just going by the `Intellisense` warning that there was a problem? The intellisense warning is safe to ignore.

As it's a `ref counted pointer`, I'd suggest changing this to `static_cast<CefRefPtr<CefRequestContext>>(requestContext)` instead. Passing in as a pointer should be safe in this instance, in general you shouldn't pass a `ref counted pointer` around as a normal `pointer`.

Reply:
Will do I will need to read up on the difference.

Reply:
https://bitbucket.org/chromiumembedded/cef/wiki/GeneralUsage.md#markdown-header-reference-counting
https://www.chromium.org/developers/smart-pointer-guidelines